### ISSUE TITLE
[SCSS] crashes when scss contains parentheses in value

### ISSUE THREAD

**Issue by llwt** [prettier/prettier#5375]
**Prettier 1.15.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEA6AzhAtnGALASygHMACYAHSlNOICcCATAWniwAcAbAQ3mck4BXLFHRJSARlIAmAGZ1SACgDMpANSkALAEoA3FQC+IADQgI7GAWjpkobnToQA7gAV7CGym4A3CExMgAEZ03GAA1rgAyuyhRMTIMHSCcKZ4MFicAOqE8OgxYHCRHgSW3iUAnsjg6DamROhwdDAuIcRY3Miy3JwNpgBW6AAeAEIh4VHcOAAyRHCd3b0gA4ORcZxwAIqCEPDzPSkgMXQNdFXoYDUB7AywmUz4yAAcAAym1xANmSHsVddwJ945qY6HAAI6CAgglrcNodJBdfamBpYAgJJIHdBrTbbXbwhYHGDcQJ3RgPJDSUyJbgEThxADC2HaVSg0CBIEEDQAKkTPAiGgYDEA)
```sh
--parser scss
```

**Input:**
```scss
.something {
  grid-template-columns: 1 2fr (3 + 4);
}
```

**Output:**
```scss
TypeError: Cannot read property 'start' of undefined
    at Object.genericPrint [as print] (https://prettier.io/lib/standalone.js:17462:62)
    at callPluginPrintFunction (https://prettier.io/lib/standalone.js:15827:18)
    at https://prettier.io/lib/standalone.js:15778:16
    at Object.printComments (https://prettier.io/lib/standalone.js:15512:17)
    at printGenerically (https://prettier.io/lib/standalone.js:15777:22)
    at FastPath.call (https://prettier.io/lib/standalone.js:15623:16)
    at Object.genericPrint [as print] (https://prettier.io/lib/standalone.js:17319:21)
    at callPluginPrintFunction (https://prettier.io/lib/standalone.js:15827:18)
    at https://prettier.io/lib/standalone.js:15778:16
    at Object.printComments (https://prettier.io/lib/standalone.js:15512:17)
```

**Expected behavior:**
Parser should accept valid scss.

**Comment by ikatyang** [comment]
The fix is available in [`1.15.2`](https://github.com/prettier/prettier/blob/master/CHANGELOG.md#1152).

**Comment by llwt** [comment]
Thanks!
